src/Type/ObjectType/MediaSize.php

```diff
@@ -64,7 +64,7 @@ public static function register_type() {
 
 							if ( ! empty( $image['ID'] ) ) {
 								$src = wp_get_attachment_image_src( absint( $image['ID'] ), $image['name'] );
-								if ( is_array( $src ) && isset( $src[0] ) && ! empty( $src[0] ) ) {
+								if ( is_array( $src ) && ! empty( $src[0] ) ) {
```

Comment:
This can be simplified ([as done in the post model](https://github.com/wp-graphql/wp-graphql/blob/14de401d563f3dec99ba40c680d0f7e73b004746/src/Model/Post.php#L821)) to `! empty( $src )`